components/Common/LanguageDropDown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { LanguageIcon } from '@heroicons/react/24/outline';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import { useIntl } from 'react-intl';
+
+import { useLocale } from '@/hooks/useLocale';
+
+import styles from './index.module.css';
+
+const DropdownMenuDemo = () => {
```

Comment:
```suggestion
const LanguageDropDown = () => {
```

the name of the component should be LanguageDropdown